Fabric.Authorization.Persistence.SqlServer/Mappers/UserMapperProfile.cs

```diff
@@ -0,0 +1,8 @@
+﻿using AutoMapper;
+
+namespace Fabric.Authorization.Persistence.SqlServer.Mappers
+{
+    public class UserMapperProfile : Profile
+    {
+    }
```

Comment:
was this intentionally left blank?

Reply:
Yes, I will remove the file.

Reply:
Actually, I was wrong here. I needed to add a constructor that configured the mapping. That is done and checked in.